sql/moz-fx-data-shared-prod/telemetry_derived/mobile_engagement_v1/metadata.yaml

```diff
@@ -0,0 +1,22 @@
+friendly_name: Mobile Engagement
+description: |-
+  Aggregated summary data about mobile Firefox engagement
+owners:
+- kwindau@mozilla.com
+labels:
+  incremental: true
+  owner1: kwindau@mozilla.com
+scheduling:
+  dag_name: bqetl_mobile_engagement_model
+bigquery:
+  time_partitioning:
+    type: day
+    field: submission_date
+    require_partition_filter: true
+    expiration_days: null
+  range_partitioning: null
+  clustering:
+    fields:
+    - country
+    - normalized_os
```

Comment:
let's cluster by app_name (normalized_os won't exist)
Also we usually have sample_id

Reply:
Do you want sample ID in the final agg table? It's not a column in that table right now

Reply:
oh I mixed it up, nevermind!

Reply:
no worries! I did add app_name to the clustering and took out normalized_os, thanks!